src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -844,6 +849,12 @@ def __init__(
         self._register_event_handler("on_bot_started")
         self._register_event_handler("on_client_ready")
 
+        self._transport = transport
+        if self._transport:
+            input_transport = self._transport.input()
+            if isinstance(input_transport, BaseInputTransport):
+                input_transport.enable_audio_in_stream_on_start(False)
```

Comment:
I think we can just do:

```
        if self._transport:
            self._transport.input().enable_audio_in_stream_on_start(False)
```

`input()` should always be a `BaseInputTransport`.

Reply:
or probably better, to make sure we have an input transport:

```
        if self._transport and self._transport.input():
            self._transport.input().enable_audio_in_stream_on_start(False)
```

Reply:
One more though... Since we are using the input transport everywhere, maybe:

```
self._input_transport = transport.input() if transport else None

if self._input_transport:
            self._input_transport.enable_audio_in_stream_on_start(False)
```

Reply:
I am going to implement your suggestion of creating a variable for `_input_transport`, so we can use it anywhere. That will already help a lot.

Reply:
Done!